src/index.ts

```diff
@@ -18,16 +30,21 @@ export function exportActorProfile({
   blockedAccounts,
   blockedDomains,
   mutedAccounts
-}: ActorProfileOptions): tar.Pack {
-  const pack: Pack = tar.pack() // pack is a stream
+}: ActorProfileOptions & { media?: File[] }): Promise<{
+  addMediaFile: (
```

Comment:
are you defining this function embedded in the other one? Extract it to be the same level.

Reply:
i have extracted addMediaFile into its own function, which takes pack and manifest as arguments